README.md

```diff
@@ -233,25 +195,120 @@ print status
   - `backoff_factor` - a backoff factor to apply between retry attempts.
 
 
+### Friendbot
+```python
+# If a friendbot endpoint is provided when creating the environment (it is provided with the TEST_ENVIRONMENT),  
+# you will be able to use the friendbot method to call a service that will create an account for you
+
+client.friendbot('address')
+```
+
+### Activate Account
+**This is the only KinClient method that requires a seed**
+```python
+client.activate_account('seed')
+```
+
+
+## Account Usage
+
+### Getting Wallet Details
+```python
+# Get the public address of my wallet account. The address is derived from the seed the account was inited with.
+address = account.get_public_address()
+```
+
+### Creating a New Account
+```python
+# create a new account prefunded with MIN_ACCOUNT_BALANCE XLM
+tx_hash = account.create_account('address')
+
+# create a new account prefunded with a specified amount of XLM.
+tx_hash = account.create_account('address', starting_balance=1000)
+
+# a text memo can also be provided:
+tx_hash = account.create_account('address', starting_balance=1000,memo_text='Account creation example')
+```
+
+### Sending Currency
+```python
+# send XLM
+tx_hash = account.send_xlm('address', 100, memo_text='order123')
+
+# send KIN
+tx_hash = account.send_kin('address', 1000, memo_text='order123')
+```
+
+### Build/Submit transactions
+While the previous methods build and send the transaction for you, there is another way to send transactions with two steps
+
+Step 1: Build the transaction
+```python
+tx = account.build_send_kin('address',100,memo_text='order123')
+```
+Step 2: Submit the transaction
+```python
+tx_hash = account.submit_transaction(tx)
+```
+
+This can be useful for some advanced use cases, since the 'build' methods return a 'kin.Transaction' object.
+The transaction object can give you the tx_hash of the transaction before sending it, and can be used to perform advanced operations such as multi-signature/multi-operations **(Not implemented yet)**
+
+**Pay attention** - Building a tx locks a channel for this specific transaction, submiting it releases that lock. However if you wish to build a tx, but decide not to submit it, make sure to release this lock with
+```python
+tx.release()
+```
+
+## Keypair
+These set of methods allow you to create new keypairs.
+
+### Create a new keypair
+```python
+from kin import Keypair
+
+my_keypair = Keypair()
+# Or, you can create a keypair from an existing seed
+my_keypair = Keypair('seed')
+```
+
+### Getting the public address from a seed
+```python
+public_address = Keypair.address_from_seed('seed')
+```
+
+### Generate a new random seed
+```python
+seed = Keypair.generate_seed()
+```
+
+### Generate a deterministic seed
+```python
+# Given the same seed and salt, the same seed will always be generated
+seed = Keypair.generate_hd_seed('seed','salt')
+```
+
+### Generate a mnemonic seed:
+**Not implemented yet**
+
 ## Limitations
```

Comment:
consider rename the title to channels , I mean it should be more clear that this is Channels explanation, inside you will explain that there's a limitation.
Also I don't see much explanation what the meaning of a channel, and does it solves this issue, it's not clear enough